Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -72,7 +77,15 @@ private static void InvokeInsertCoin()
 
         }
 
-        // optional InitOptions
+        [MonoPInvokeCallback(typeof(Action<string>))]
+        private static void InvokeOnErrorInsertCoin(string error)
+        {
+            onError?.Invoke(error);
```

Comment:
If onError is not provided, perhaps just throw an Exception (it would be the developer's responsibility to manage it)